source/content/certification/exam-faq.md

```diff
@@ -0,0 +1,129 @@
+
+4. **Tablets, Chromebooks, iPads, and mobile devices are not supported.**
+
+Remote exam sessions must be completed on a laptop or desktop computer that meets the minimum technical requirements.
+
+
+</Accordion>
+
+
+<Accordion title="Do I need a camera" id="" icon="wrench">
+In order to take the exam with our automated proctoring service, you will need a working web camera to record your test taking session. Please reach out to us if you have any questions about this.
+</Accordion>
+
+
+<Accordion title="What are the technical requirements for taking a Pantheon Certification?
+" id="" icon="wrench">
+
+
+
+In order to maintain compatibility with our online proctoring service, we require that exam takers use machines that adhere to the following minimum technical specifications:
+
+**Computer:** We support most standard laptops and desktops that include the minimum hardware requirements listed below. Chromebooks, Linux, iPads, tablets, and mobile devices are not supported.
+
+**Operating System:** We support environments that use Windows 10 or greater, or macOS 10.12 (Sierra) or greater. Unfortunately, our online proctoring service does not support the use of Linux machines. If you are using a Linux machine, we are happy to accommodate by setting up a live proctored test that does not use the automated proctoring service. Windows S mode is not supported.
+
+**Hardware Requirements:** The following are required: at least 1 GB available disk space, minimum 2GB RAM, and Intel Quad-core 1.8GHz CPU (or better) or AMD Quad-core 2.0 GHz CPU (or better).
+
+**Web Browser:** We Support Chrome, Firefox, Safari, Microsoft Edge, and Internet Explorer 11+. Brave and Opera are not supported at this time, nor is Microsoft Edge for macOS.
+
+**Internet Speed:** Minimum upload speed of 1 Mbps, and a minimum download speed of 2 Mbps. Tethering and hotspots are not supported. (An active broadband internet connection is assumed since the examinations are done online).
+
+**Web Camera & Microphone:** An external or internal web camera and microphone are required. Camera and microphone drivers must have been updated or released within the last 5 years. 3D Webcams are not supported at this time.
+
+</Accordion>
+<Accordion title="How do I register for an exam?" id="" icon="wrench">
+
+
+You can register for the Pantheon WebOps Certification Exam through our Certification Website. On the homepage of the site, there is a form for you to fill out. It will ask for your name, email address, organization (if applicable), and some technical information about the computer you will be using to take the exam. This is to help us ensure that your computer is technically compatible with our exam engine and proctoring service. You will then choose a date to take the exam. Once you submit this form, you will be redirected to a payment processor to complete your payment and secure your spot in our certification program.
```

Comment:
@stevector suggest updating the answer here so that it links to the external Certification site (instead of existing copy which says "On the homepage of the site,"..)

Reply:
@pantheonsteve is this info still accurate ^ ?